<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrecoEntrega</class>
 <widget class="QDialog" name="PrecoEntrega">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_procurar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Procurar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_entrega">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>551</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_procurabairro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Entre com bairro ou rua&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_extra">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_bairro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>[bairro]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cidade">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>[cidade]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor: R$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Percentual:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Taxa Extra: R$</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_percentual">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string extracomment="%"/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_valor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string extracomment="R$"/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gravar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gravar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
